contextPlus.js

```diff
@@ -43,14 +44,16 @@ if (browser.contextualIdentities !== undefined) {
             windowId
           } = tab;
 
-          browser.tabs.create({
+          const newTabPromise = browser.tabs.create({
             active,
             cookieStoreId,
-            index,
+            index: index + (moveTab ? 0 : 1),
             pinned,
             url,
             windowId
-          }).then(() => browser.tabs.remove(tab.id));
+          });
+          if(moveTab)
```

Comment:
A minor nitpick--the formatting of this code should align with the rest of the file. This means it would look like

```javascript
if (moveTab) {
  newTabPromise.then(() => browser.tabs.remove(tab.id));
}
```